ogr/services/github.py

```diff
@@ -209,6 +209,8 @@ def get_releases(self) -> List[Release]:
 
 
 class GithubUser(BaseGitUser):
+    service: GithubService
+
```

Comment:
previously the class looked like:
```python
class GithubUser(BaseGitUser):
    def __init__(self, service: GithubService) -> None:
        super().__init__(service=service)

    def get_username(self) -> str:
        return self.service.github.get_user().login
```
and mypy kept telling me that:
`error: "GitService" has no attribute "github"`
while pointing at the
`return self.service.github.get_user().login`
line.

I resolved it by declaring this instance variable.
@lachmanfrantisek do you have any better idea?

Reply:
Isn't this a class attribute or is this used only for the typing?

Reply:
It's just hint/annotation, see https://www.python.org/dev/peps/pep-0526/#class-and-instance-variable-annotations